e2e/e2e_docker_test.go

```diff
@@ -22,15 +22,19 @@ import (
 	"github.com/GoogleContainerTools/kpt/pkg/test/runner"
 )
 
-func TestPipeline(t *testing.T) {
-	runPipelineTests(t, "../internal/pipeline/testdata/")
+func TestFnRender(t *testing.T) {
+	runTests(t, "../internal/pipeline/testdata/")
```

Comment:
does this only work on POSIX systems?

Obligatory comment about being careful about OS-specific paths

Dittor for any use of paths here: ExecPath, FnPath.

Reply:
I added OS support to test cases sheet.

Reply:
It's easy to make it portable by using `filepath.Join("..", "internal", "pipeline", "testdata")`, so let's do that.

Reply:
done